views/includes/page-templates/beneficial-owner-nature-of-control-types.html

```diff
@@ -11,7 +9,7 @@
 {% set errorMessage = errors.beneficial_owner_nature_of_control_types if errors %}
 {% include "includes/inputs/nature-of-control-checkboxes/beneficial-owner-nature-of-control-types.html" %}
 
-{% if isBeneficialOwnerOther %}
+{% if not isBeneficialOwnerGov %}
```

Comment:
Does this change registration?

Reply:
All looks good. Screenshot of register journey (locally) for adding a Gov BO, nature of control questions are as expected (none of the trustees boxes)

<img width="1715" alt="image" src="https://user-images.githubusercontent.com/120723588/230411724-e9f5aa84-274c-4e46-a36f-51a4d1c7f130.png">

Reply:
Individual for comparison. 
![image](https://user-images.githubusercontent.com/120723588/230412699-7a78c22e-2e2a-44f5-b341-c42241500b33.png)

Reply:
For further clarification, Matthew sent me this message: "We should only be asking for the natures of control that come under **'The beneficial owner:**' & '**The beneficial owner is a member of a firm that is not a legal person under its governing law, and:**' for GOV BOs"